aireview/poetry.lock

```diff
@@ -1,5 +1,188 @@
 # This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
 
```

Comment:
**documentation**: Ensure poetry.lock changes are accompanied by corresponding pyproject.toml changes to maintain dependency documentation. Consider adding a comment in the commit message explaining the reason for dependency updates.